src/shared/components/ResourceDetails/ResourceDetails.scss

```diff
@@ -24,7 +24,7 @@
   min-width: 200px;
 }
 
-@container (min-width: 650px) {
```

Comment:
Search for the rest of `@container` and check if it's working, there is 16 results in 10 files